core/src/main/scala/cats/Traverse.scala

```diff
@@ -37,6 +38,7 @@ import simulacrum.typeclass
  *
  * See: [[https://www.cs.ox.ac.uk/jeremy.gibbons/publications/iterator.pdf The Essence of the Iterator Pattern]]
  */
+@implicitNotFound("Could not find an instance of Traverse for ${F}")
```

Comment:
We removed these in https://github.com/typelevel/cats/pull/4060. Michael is right, we should probably stop using simulacrum scalafix :(